terraform/service_export.tf

```diff
@@ -85,28 +94,20 @@ resource "google_cloud_run_service" "export" {
           name  = "EXPORT_BUCKET"
           value = google_storage_bucket.export.name
         }
-
-        dynamic "env" {
-          for_each = local.common_cloudrun_env_vars
-          content {
-            name  = env.value["name"]
-            value = env.value["value"]
-          }
-        }
       }
     }
 
     metadata {
       annotations = {
-        "autoscaling.knative.dev/maxScale" : "1000",
-        "run.googleapis.com/cloudsql-instances" : google_sql_database_instance.db-inst.connection_name
+        "autoscaling.knative.dev/maxScale" : "2",
```

Comment:
2 might be too low here.

I think we should perhaps turn down the max concurrency (10) and increase the max scale also (10)

it's really for extreme cases, "catch up" or massive tenancy.

Reply:
Done